src/views/LoginView.tsx

```diff
@@ -73,6 +82,11 @@ export default function LoginView() {
         >
           Login
         </Button>
+        {config.oidcEnabled && (
+          <Link href={getOidcLoginUrl()} sx={{ 'text-align': 'center' }} underline="hover">
```

Comment:
```suggestion
          <Link href={getOidcLoginUrl()} textAlign="left" variant="body2" underline="hover">
```

Reply:
In the global css file the font-size is declared for all HTML elements. I will remove this in a next PR, so to add a font-family you'll need to add a link variant. Also almost all css props are availible without using the `sx` props